index.html

```diff
@@ -156,22 +156,21 @@ <h2>Food, Friends, and Fun</h2>
             <div>
                 <h1>Event Schedule</h1>
                 <table class="schedule">
```

Comment:
My merge commit nuked the div ID, sorry.
```suggestion
                <table class="schedule" id="scheduletable">
```